showcase/src/app/app.module.ts

```diff
@@ -154,7 +154,11 @@ export const metaReducers: MetaReducer<State>[] = ENV !== "production" ? [logger
 		StarkRoutingModule.forRoot(),
 		SharedModule,
 		DemoModule,
-		StarkAppLogoModule
+		StarkAppLogoModule,
```

Comment:
All these modules (except the StarkAppLogoModule) should not be imported here, they are now imported in the Demo module ;)